task/task.go

```diff
@@ -0,0 +1,10 @@
+package task
+
+type UserTaskStatus int8
+
+const (
+	USER_TASK_STATUS_UNSPECIFIED = iota
+
+	USER_TASK_STATUS_IN_PROGRESS
+	USER_TASK_STATUS_FINISHED
```

Comment:
Kalo enum di Go harusnya yang waras aja, nggak perlu ngikutin punyanya Protobuf

```suggestion
	UserTaskStatusUnspecified UserTaskStatus = iota

	UserTaskStatusInProgress
	UserTaskStatusFinished
```

Reply:
lebih suka kasih string ketimbang gini ;D

Reply:
> lebih suka kasih string ketimbang gini ;D

ngikutin schema db yg lain pakai `int` soale :D

Reply:
Punten ini gmn ya jadinya @aldy505